### PR TITLE
html-rec

### PR DESCRIPTION
**Pull request by aghaynes**
recruitment report

### CODE REVIEW

File: template.qmd [hunk 1]
```diff
@@ -0,0 +1,114 @@
+---
+title: "CTU project name and CTU project number"
+subtitle: "Recruitment report"
+description: "date"
```
Comment (MoserGitHub) [inline template.qmd]:
```suggestion
description: "Study protocol version"
```

File: template.qmd [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  theme_classic() +
+  scale_x_date(labels = function(x) format(x, format = "%d %b %Y")) +
+  scale_color_manual(values = cols) +
+  theme(legend.position = "bottom")
+p2 <- plot(acc, which = "abs", unit = "month", engine = "gg") +
+  theme_classic() +
+  scale_x_date(labels = function(x) format(x, format = "%b %Y")) +
+  scale_fill_manual(values = cols) +
+  theme(legend.position = "bottom")
+p1 + p2
```
Comment (MoserGitHub) [inline template.qmd:102]:
```suggestion
p1 + p2

library(DT)
report_table <- summary(acc)
names(report_table) <- report_table[1,]
report_table <- report_table[-1,]
report_table$Center <- as.factor(report_table$Center)
datatable(report_table, 
          rownames = F, filter = 'top', 
          options = list(pageLength = 20))
```

Comment (MoserGitHub) [reply template.qmd:102]: Add interactive table as example

File: template.qmd (hunk 1 above)
Comment (MoserGitHub) [reply template.qmd]: Date is already in header of report. Maybe study protocol version?

File: template.qmd [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+#| echo: false
+#| warning: false
+#| fig.height: 3
+plot(acc, engine = "gg", which = "pred", target = 400) +
+  theme_classic() +
+  scale_x_date(labels = function(x) format(x, format = "%d %b %Y")) +
+  theme(plot.title.position = "plot")
+```
+
+
```
Comment (MoserGitHub) [inline template.qmd:127]: Same info as in html report about report history, reproducibility and session info

File: template.qmd [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+author:
+  - name: "Quarto Muster"
+    orcid: 111
+    email: "xxx@ctu.unibe.ch"
+    affiliation: "CTU Bern, University of Bern"
+format: qmd-ctuhtml-html
+bibliography: references.bib
+---
+
+
```
Comment (MoserGitHub) [inline template.qmd:14]:
```suggestion
# Overview

- Number of planned centers.
- Planned final sample size.
- Planned interim analyses.
- Study protocol version (maybe link to published study protocol)
```

**Review by MoserGitHub — COMMENTED** [review]
Hi Alan. Here some suggestions for changes.

File: template.qmd (hunk 1 above)
Comment (aghaynes) [reply template.qmd]: the protocol version isn't relevant/important for a recruitment report... maybe we just remove it all together?

**Review by MoserGitHub — APPROVED** [review]
As discussed

**Comment by aghaynes** [comment]
thanks @MoserGitHub

**Pull request merged**